### PR TITLE
WorkflowDetailFragment Improvements

### PR DESCRIPTION
**Pull request by Hiteshgautam01**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the checks with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Hiteshgautam01** [comment]
<h1 align= "center" >UI after changes</h1>
<h1 align="center">

**Comment by sagar15795** [comment]
@Hiteshgautam01  could you please mention the changes also?

**Comment by Hiteshgautam01** [comment]
Hey @sagar15795, the uploader image is now using circular image view.
The web view for description is in card view.
The title is shifted up using relative layout instead of linear layout.
Also some improvements in margin and paddings.

File: app/src/main/res/layout/fragment_detail_workflow.xml [hunk 1]
```diff
@@ -102,7 +111,8 @@
                             android:layout_width="wrap_content"
                             android:layout_height="wrap_content"
                             android:textColor="@android:color/black"
-                            android:textSize="@dimen/tv_uploader_name_text_size"/>
+                            android:textSize="@dimen/tv_uploader_name_text_size"
+                            android:layout_marginTop="4sp"/>
```
Comment (sagar15795) [inline app/src/main/res/layout/fragment_detail_workflow.xml]: use `4dp` in place of `4sp`. and also move it to `dimens.xml`

**Comment by Hiteshgautam01** [comment]
Hey @sagar15795, Done the requested changes.

**Comment by sagar15795** [comment]
@Hiteshgautam01 If the workflow name length is full filled then I think it will come under the fab icon. I think we should move workflow name below the fab icon.

**Comment by Hiteshgautam01** [comment]
No @sagar15795, it will not come under the fab icon because of the right margin I have added in the title and I test for that case also.  It is working fine.

**Comment by sagar15795** [comment]
instead of margin make it left to fab icon using relative layout property because margin depends on screen. It might come under it.

**Comment by Hiteshgautam01** [comment]
Hey @sagar15795, Now I have used the `android:layout_toLeftOf=` which is working perfectly.

**Pull request merged**